js/util/templateHelpers.js

```diff
@@ -36,6 +36,91 @@ define(["globals", "lib/faces", "lib/knockout", "util/helpers"], function (g, fa
         }
     };
 
+    ko.bindingHandlers.letterGrade = {
+      update: function(element, valueAccessor) {
+        var arg, output;
+        arg = ko.unwrap(valueAccessor());
+
+        if (arg < 0.20) {
```

Comment:
zscore! That would require precomputing and caching them, though. If we could get away with something simpler like this, that'd be great. Will require some more testing to see what cutoffs make sense.

>  If we do something like that though, would we also want to do something to normalize the displayed base ratings?

No, base ratings are base ratings. If we want to rescale them, that should happen before they are saved.

And I'm still leaning towards this type of UI, plus or minus a couple categories. The other one is too heavy. But I don't want to merge this until I test it some more.